tests/test_cli.py

```diff
@@ -545,8 +546,18 @@ def test_refactor_product_should_not_warning_if_comment(tmp_empty):
 ])
 def test_test_notebook_runs(tmp_empty, code, output):
     nb_ = jupytext.reads(code, fmt='py:light')
-    for filename in ['nb.ipynb', 'nb.py']:
-        jupytext.write(nb_, filename)
-        runner = CliRunner()
-        result = runner.invoke(cli.test, filename)
-        assert output in result.output
+    filenames = ['nb.ipynb', 'nb.py']
```

Comment:
I pushed some changes to the error messages, please update the tests

Reply:
Updated